azkaban-common/src/main/java/azkaban/executor/ExecutionControllerUtils.java

```diff
@@ -100,11 +108,36 @@ public static void finalizeFlow(final ExecutorLoader executorLoader, final Alert
       logger.error("Failed to finalize flow " + flow.getExecutionId() + ", do not alert user.", e);
     }
 
+    // If the flow is in state EXECUTION_STOPPED and user enabled restartability in flow
+    // parameters, invoke restart callback
+    if (flow.getStatus() == Status.EXECUTION_STOPPED) {
+      final ExecutionOptions options = flow.getExecutionOptions();
+      if (options != null && !options.isExecutionRetried()) {
+        final Map<String, String> flowParam = options.getFlowParameters();
```

Comment:
extreme nitpick : Consider renaming to flowParams.

Reply:
changed to flowParams